### PR TITLE
Add FAQ's page with styles

### LINKED ISSUE #19: FAQs page
# REQUIREMENTS
We need to create a FAQs page to help the new users to understand what is Let Me Help You

# DESIGN
image

# Acceptance Criteria
List all required acceptance criteria
- [x] A page with FAQs
- [x] Responsive

### PR DESCRIPTION
**Pull request by SampabloC**
# Description
What did you implemented/Why did you implemented this:
 - Add FAQ's view and implement styles
 - Made responsive FAQ's view
 - Add internationalization for English and Spanish on FAQ's view and navbar

Resolve #19 

  ## Screenshots 
Desktop English Version
imagen

Deskptop Spanish Version
imagen

Mobile Enlgish Version
imagen

Mobile Spanish Version
imagen

### CODE REVIEW

**Review by brayandiazc — DISMISSED** [review]

**Review by OswaldoPineda — DISMISSED** [review]

File: app/views/home/faqs.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="faqs-text">
+      <h3>How long is an appointment?</h3>
+      <p>
+      Appointments last 50 minutes. <br>
+      It is important that you enter your appointment on time. The consultation times have been programmed especially for you, if you are late for your appointment, it will have to be shorter to respect the schedules of the other patients. <br>
+      Regardless of the duration of your appointment, it will be charged in full. 
+      </p>
+    </div>
+  </section>
+</main>
```
Comment (YilKanda) [inline app/views/home/faqs.html.erb:40]: I suggest you use internationalization, so you can add the FAQ in multiple languages.

[Rails Internationalization (I18n)](https://guides.rubyonrails.org/i18n.html)

**Review by brayandiazc — APPROVED** [review]
They should think of a better way to structure the translations.
As they are doing it, it works, but in my case I prefer to anchor the translations to several files "es.name_controller.yml" in this way as the application grows and words must be corrected or added is easier

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by AlejandroVI99 — APPROVED** [review]
Nice! Keep going!

**Pull request merged**